src/components/radio-input.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { Fragment } from 'react';
+import { FormFieldErrorList } from './form-field-error-list';
+
+interface RadioInputProps {
+  name: string;
+  label: string;
+  value: string | undefined;
+  onValueChange: (value: string) => void;
+  checked?: boolean;
+  errors?: string[];
+}
+
+export const RadioInput = ({
+  name,
+  label,
+  value,
+  onValueChange,
+  checked,
+  errors,
+}: RadioInputProps): React.ReactElement => {
+  return (
+    <Fragment>
+      <input name={name} type='radio' value={value} onChange={(e) => onValueChange(e.target.value)} checked={checked} />
+      {label}
```

Comment:
esse label aqui ta sendo colocado direto sem nenhuma tag né, acho melhor encapsular uma tag aqui, não é muito interessante deixar conteudo livre direto no body sem um "container"

Reply:
Não entendi muito bem a ideia. Seria tirar o label do RadioInput e adicionar o label quanto o componente for utilizado? Algo como isso?
```javascript
...
<RadioInput props... />
<p>{label}</p>
...
```

Reply:
Ah, nvm, eu tinha enxergado outra coisa. Acho que o mais legal aqui seria que essa label fosse uma tipografia, mas como é uma label, especificamente a tag [label](https://www.w3schools.com/tags/tag_label.asp) pode ser a mais indicada pra isso.